prometheus/linux/crunchy-prometheus.yml

```diff
@@ -46,7 +46,7 @@ scrape_configs:
   ## Monitoring for tcp services that don't have an associated exporter can be accomplished using the tcp probe 
   ##   of the blackbox_exporter provided by the Prometheus developers.
   ## Note this only provides a simple up/down that the service is listening on the given IP/port.
-  ## If an exporter is available, it is advised to use an Absence alert vs the blackbox probe.  
+  ## If an exporter is available, you can either use a blackbox probe or an Absense alert for each node
```

Comment:
I don't follow. We start by saying 'if you don't have an exporter use blackbox' and then we say 'but if you do, use blackbox or an absence alert' .. should we say 'if you do, use that exporter'? or simply remove the 'if you do' section and make this just about what do when there isn't one?

Reply:
So, while the blackbox probe is more vague compared to an absence alert from the exporter itself, it is easier to set up. An absence alert needs a separate alert to be set for each individual host running an exporter. Blackbox is just a single line added to the prometheus config for each host to monitor. Thought it might make sense to leave it as a user preference, but if it's less confusing to recommend one over the other, i'm fine leaving the old description.

Reply:
Basically, if the exporter is available, you could use either to monitor for the exporter/service going away. If there is no exporter available, you must use blackbox

Reply:
Updated this comment to be clearer that an absence alert is preferred

Reply:
Any preferred one between two ?

Reply:
See the updated line. It notes a preferred one. Not sure why it's not showing up here.